app/uk/gov/hmrc/identityverificationquestions/config/AppConfig.scala

```diff
@@ -35,6 +35,12 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   def allowedUserAgentList: Seq[String] = getStringList("allowedUserAgentList").getOrElse(Seq.empty[String])
 
+  def deniedUserAgentListForP60: Seq[String] = getStringList("deniedUserAgentListForPaye").getOrElse(Seq.empty[String])
```

Comment:
typo in config, it should be deniedUserAgentListForP60 instead of deniedUserAgentListForPaye